### DISCUSSION TITLE
gobang now available as a NetBSD package

### DISCUSSION THREAD

**Discussion by 0323pin** [TaKO8Ki/gobang#116] [General]
FYI, I've packaged `gobang` for NetBSD and merged it into the main branch.
https://pkgsrc.se/databases/gobang

Regards,
pin

**Comment by TaKO8Ki** [comment]
Thank you! That's nice. I will add the way to install gobang on NetBSD to README.

**Reply by TaKO8Ki** [reply]
I've added the way to install gobang on NetBSD to README.

**Reply by 0323pin** [reply]
Thanks! I think you could also add it here, if you want.

> Cross-platform support (macOS, Windows, Linux)
